### DISCUSSION TITLE
The requested module './useSuspenseQueries.js' does not provide an export named 'useSuspenseQueries'

### DISCUSSION THREAD

**Discussion by kkMihai** [TanStack/query#6475] [Q&A]
```
The requested module './useSuspenseQueries.js' does not provide an export named 'useSuspenseQueries'
```

**Environment:**
- Node.js Version: 20.10.0
- Next.js Version: 14+
- React-Query Version: 5.8.9

Might be a compatibility issue but i m not sure

![image](https://github.com/TanStack/query/assets/59262001/9621b09f-f3df-4052-ac76-2a27c3e495ee)

**Comment by TkDodo** [comment]
need to see more than an image please, like a full codesandbox reproduction
